index.js

```diff
@@ -100,23 +128,31 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
+ 
+function triple(value) {
+  console.log(value * 3)
+}
```

Comment:
On the right track! This function should return instead of console.logging because myMap function needs a return value to push to the new array it's making. Right now, the triple function will return undefined

```javascript
function triple(value) {
  console.log(value * 3)
}
```
_______________________________
See below for an example of using console.log vs return

```javascript
function addTwoNums(num1, num2) {
     return num1 + num2
}

let sum = addTwoNums(10, 20) // value of sum will be 30
```


VS

```javascript
function addTwoNums(num1, num2) {
     console.log(num1 + num2)
}

let sum = addTwoNums(10, 20) // value of sum will be undefined
```